test/fixtures/output/node/fetch/cookies.js

```diff
@@ -1,7 +1,7 @@
 const fetch = require('node-fetch');
 
 const url = 'http://mockbin.com/har';
-const options = {method: 'POST', headers: {cookie: 'foo=bar; bar=baz; '}};
```

Comment:
Cool side effect of this work is now cookies for `node-fetch` won't have an unnecessary space at the end.